typings/index.d.ts

```diff
@@ -433,47 +459,57 @@ declare module 'enmap' {
      * Can be a path with dot notation, such as "prop1.subprop2.subprop3"
      * @returns The enmap.
      */
-    public delete(key: K, path?: string): this;
+    public delete(key: string, path?: string): this;
 
     /**
      * Delete a property from an object or array value in Enmap.
      * @param key Required. The key of the element to delete the property from in Enmap.
      * @param path Required. The name of the property to remove from the object.
      * Can be a path with dot notation, such as "prop1.subprop2.subprop3"
      */
-    public deleteProp(key: K, path: string): void;
+    public deleteProp(key: string, path: string): void;
 
     /**
      * Deletes everything from the enmap. If persistent, clears the database of all its data for this table.
      */
     public deleteAll(): void;
 
+    /**
+     * Completely destroys the entire enmap. This deletes the database tables entirely.
+     * It will not affect other enmap data in the same database, however.
```

Comment:
Why the comma?

Reply:
That's how grammar works. However is a connective that has a comma before and after it

Reply:
I dunno. I just copied the description from the code. I am really not the best person to ask about grammar stuff. Just edit it to your likings :)

Reply:
> That's how grammar works. However is a connective that has a comma before and after it

Ah you're right, it's an interrupter in this case.